ntoskrnl/inbv/inbv.c

```diff
@@ -920,8 +930,14 @@ DisplayBootBitmap(IN BOOLEAN TextMode)
 #endif
 
 #ifdef INBV_ROTBAR_IMPLEMENTED
+        /* Save previous screen pixels to buffer */
+        InbvScreenToBufferBlt(Buffer, 0, 0, 22, 9, 24);
```

Comment:
You probably can move this whole block inside the `if (Bar)` since saving restoring is unnecessary otherwise?